tcms/testruns/tests/test_admin.py

```diff
@@ -91,3 +91,26 @@ def test_changelist_view_icon_column(self):
             """
             ),
         )
+class TestTestExecutionStatusDelete(LoggedInTestCase):
```

Comment:
There's already a class for TestExecutionStatusAdmin tests so all of the below should go inside of it. However that's unrelated to the actual test scenario below.